WebAssembly/Runtime/UnmanagedMemory.cs

```diff
@@ -16,8 +17,90 @@ public sealed class UnmanagedMemory : IDisposable
-        private static readonly RegeneratingWeakReference<long[]> emptyPage = new RegeneratingWeakReference<long[]>(() => new long[Memory.PageSize / 8]);
 
+        /// <summary>
+        /// Copies a <paramref name="length"/> of bytes to the <paramref name="destination"/> memory location from the <paramref name="source"/> memory location.
+        /// </summary>
+        /// <param name="destination">The write target memory location.</param>
+        /// <param name="source">The read target memory location.</param>
+        /// <param name="length">The count of bytes to be copied.</param>
+        public delegate void CopyBlockDelegate(IntPtr destination, IntPtr source, uint length);
+        
+        /// <summary>
+        /// Copies a length of bytes to the destination memory location from the source memory location.
+        /// See <see cref="CopyBlockDelegate"/> for details.
+        /// </summary>
+        public static readonly CopyBlockDelegate CopyBlock;
+
+        /// <summary>
+        /// Sets a <paramref name="length"/> of bytes to the given <paramref name="value"/> at a <paramref name="destination"/> memory location.
+        /// </summary>
+        /// <param name="destination">The write target memory location.</param>
+        /// <param name="value">The byte value to use.</param>
+        /// <param name="length">The count of bytes to be copied.</param>
+        public delegate void InitBlockDelegate(IntPtr destination, byte value, uint length);
+
+        /// <summary>
+        /// Sets a length of bytes to the given value at a destination memory location.
+        /// See <see cref="InitBlockDelegate"/> for details.
+        /// </summary>
+        public static readonly InitBlockDelegate InitBlock;
+	
+        static UnmanagedMemory()
+        {
+            const string dynamicModuleName = nameof(UnmanagedMemory) + "DynamicMethods";
+            
+            var module = AssemblyBuilder.DefineDynamicAssembly(
+                        new AssemblyName(dynamicModuleName),
+                        AssemblyBuilderAccess.RunAndCollect
+                    )
+                    .DefineDynamicModule(dynamicModuleName);
+
+            var dynClass = module.DefineType(
+                "DynamicMethods",
+                TypeAttributes.Public | TypeAttributes.Abstract | TypeAttributes.Sealed
+            );
+            
+            var initBlock = dynClass.DefineMethod(
+                nameof(InitBlock),
+                MethodAttributes.Public | MethodAttributes.Static | MethodAttributes.Final,
+                CallingConventions.Standard,
+                typeof(void), new []{typeof(IntPtr), typeof(byte), typeof(uint)});
+            initBlock.SetImplementationFlags(MethodImplAttributes.AggressiveInlining);
+            {
+                var il = initBlock.GetILGenerator();
+                il.Emit(OpCodes.Ldarg_0);
+                il.Emit(OpCodes.Ldarg_1);
```

Comment:
The passed value is always `0`, so we can use `OpCodes.Ldc_I4_0` and remove the parameter.